charts/kueue/values.yaml

```diff
@@ -0,0 +1,73 @@
+# Default values for kueue.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
```

Comment:
```suggestion
nameOverride: ""
fullnameOverride: ""
namespaceOverride: ""
```

Reply:
Thanks! fixed it
About namespaceOverride, I can't find it in [helm's repo](https://github.com/helm/helm), so I didn't add it.